app/assets/javascripts/patients.coffee

```diff
@@ -23,6 +23,23 @@ valueToNumber = (val) ->
   +val || 0
 
 ready = ->
+  fields = [
+    '#patient_fulfillment_procedure_cost'
+    '#patient_fulfillment_check_number'
+    '#patient_fulfillment_gestation_at_procedure'
+    '#patient_fulfillment_date_of_check'
+    '#patient_fulfillment_procedure_date'
+  ]
+  i = 0
+  field_selectors = ''
+
+  while i<fields.length
```

Comment:
Sure! basically ```field_selectors``` looks something like this

```'#patient_fulfillment_procedure_cost, #patient_fulfillment_check_number, #patient_fulfillment_gestation_at_procedure, ...'```

This just programmatically concats the fields for less mistyping. And I can feed this variable to the onChange jquery handler without too many problems.